tests/platform_tests/test_intf_fec.py

```diff
@@ -91,3 +87,51 @@ def test_config_fec_oper_mode(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
 
             if not (fec == "rs"):
                 pytest.fail("FEC status is not restored for interface {}".format(intf['interface']))
+
+
+def get_interface_speed(duthost, interface_name):
+    """
+    Get the speed of a specific interface on the DUT.
+
+    :param duthost: The DUT host object.
+    :param interface_name: The name of the interface.
+    :return: The speed of the interface as a string.
+    """
+    logging.info(f"Getting speed for interface {interface_name}")
+    intf_status = duthost.show_and_parse("show interfaces status {}".format(interface_name))
+
+    speed = intf_status[0].get('speed')
+    logging.info(f"Interface {interface_name} has speed {speed}")
+    return speed
+
+    pytest.fail(f"Interface {interface_name} not found")
+
+
+def test_verify_fec_stats_counters(duthosts, enum_rand_one_per_hwsku_frontend_hostname,
```

Comment:
@vvolam The test should fail if there are uncorrectable FEC errors

Reply:
@prgeor, Extended the test for the same. Thanks!